fe-vite-react/src/App.jsx

```diff
@@ -1,34 +1,40 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import { useState} from 'react'
+import {Route, Routes, Link} from "react-router-dom";
+import NewsList from './NewsList';
+import SingleArticlePage from './SingleArticlePage';
+
+
+
+const Header = () => {
+  return (
+    <div className='header'>
+  <h1 className='title'>Latest NC News </h1>
+  </div>
+  )
+};
```

Comment:
if we're extracting the Header out of App you may as well make another component file for it to neaten and de-bloat this file